packages/identitykit/src/lib/identity-kit.ts

```diff
@@ -33,5 +33,7 @@ export class IdentityKit {
         certified: false,
       })
     ).toString()
+
+    return Number(balance) / 10 ** 8
```

Comment:
maybe put const E8S?

Reply:
knew you will point to that 😅 decided E8S is not clearer than 10 ** 8, and is only used once now